cpp/log/ct_extensions.h

```diff
@@ -10,12 +10,12 @@ void LoadCtExtensions();
 
 // Defines structure for redacted label count to hold a stack of integers
 typedef struct RedactedLabelCount_st {
-  STACK_OF(ASN1_INTEGER)* redacted_labels;
+  STACK_OF(ASN1_INTEGER) * redacted_labels;
```

Comment:
I guess we should look at replacing a lot of this with the new scoped ssl pointers and, if we're lucky, avoid this mess that way :)

I'll leave this as-is for now though...